.github/dependabot.yml

```diff
@@ -29,3 +29,29 @@ updates:
     open-pull-requests-limit: 10
     labels:
       - dependencies
+    groups:
+      gomod-all:
+        patterns:
+          - "*"
+  - package-ecosystem: gomod
+    directory: "/test/docker-e2e"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+    labels:
+      - dependencies
+    groups:
+      gomod-all:
+        patterns:
+          - "*"
+  - package-ecosystem: gomod
+    directory: "/test/interchain"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+    labels:
+      - dependencies
+    groups:
+      gomod-all:
+        patterns:
+          - "*"
```

Comment:
nice! do we need to include the go mod at the root of the directory as well? or what is the reasoning for keeping that separate?